src/tests/integration/CMakeLists.txt

```diff
@@ -0,0 +1,45 @@
+#//////////////////////////////////////////////////////////////////////////////////////
+#// This file is distributed under the University of Illinois/NCSA Open Source License.
+#// See LICENSE file in top directory for details.
+#//
+#// Copyright (c) 2021 QMCPACK developers.
+#//
+#// File developed by: Peter Doak, doakpw@ornl.gov, Oak Ridge National Laboratory
+#//
+#// File created by: Peter Doak, doakpw@ornl.gov, Oak Ridge National Laboratory
+#//////////////////////////////////////////////////////////////////////////////////////
+
+# This test is always built and works due to openmp pragmas becoming no ops
+# even in versions of the code that do not use OMPAllocator.
+
+set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${QMCPACK_UNIT_TEST_DIR})
+
+set(SRC_DIR dualallocators)
+set(UTEST_DIR ${qmcpack_BINARY_DIR}/tests/integration)
+
+execute_process(COMMAND ${CMAKE_COMMAND} -E make_directory "${UTEST_DIR}")
+
+
+set(UTEST_EXE test_${SRC_DIR})
+set(UTEST_NAME deterministic-integration_${SRC_DIR})
```

Comment:
Just leaving a note. I notice on develop.
```
786/816 Test   #31: deterministic-unit_test_hamiltonian_force ........................................................   Passed    1.42 sec
        Start   26: deterministic-unit_test_wavefunction_jastrow
787/816 Test   #26: deterministic-unit_test_wavefunction_jastrow .....................................................   Passed    1.34 sec
        Start    3: deterministic-integration_dualallocators
788/816 Test    #3: deterministic-integration_dualallocators .........................................................   Passed    0.58 sec
```
better to rename as `deterministic-unit_test_containers`

Reply:
See #3318